### PR TITLE
adding loder to website

### PR DESCRIPTION
**Pull request by ananya-agrawal**
https://ananya-agrawal.github.io/Aparoksha-18/register/index.html

### CODE REVIEW

**Comment by ananya-agrawal** [comment]
https://codepen.io/JorenVanHee/pen/ycodJ?page=3&

File: register/index.html [hunk 1]
```diff
@@ -16,7 +16,11 @@
 </head>
 
 <body>
-
+  .tetris
+ .block1
+ .block2
+ .block3
+ .block4
```
Comment (shreyanshdwivedi) [inline register/index.html]: If you are using https://codepen.io/JorenVanHee/pen/ycodJ?page=3& as a reference, use the compiled HTML, which you can find in dropdown of HTML section. Replace this code with the compiled one.

**Comment by pradeepgangwar** [comment]
@ananya-agrawal Place this code at the end of file.
and replace .hameid .... with the class name of div of loader. 
```
<script>
    jQuery(window).load(function(){
        jQuery(".hameid-loader-overlay").fadeOut(500);
    });
</script>
```

**Comment by pradeepgangwar** [comment]
@ananya-agrawal Make a file register/js/loader.js and place code like this there:

```
$(document).ready(function($) {
    // make loader of full size here.
});
$(window).on('load', function () {
    $('.tetris').fadeOut();
    //make site visible again
});
```
Add this file in scripts in index.html
You will see that tetris is removed after page is loaded.

Remove comments and write some code that adds class to body to hide it and make loader of full width and then downside write function to make body visible again.

Take reference from a opencode PR itself. https://github.com/opencode18/OpenCode-Collaborative/pull/207/files If you don't understand from there I suggest watching some tutorial online on seek help from any one of the mentor of this repo.

**Comment by pradeepgangwar** [comment]
@ananya-agrawal I made some changes to PR. Please go through code once again.

**Comment by pradeepgangwar** [comment]
@ananya-agrawal We don't want this loader. Please suggest a new one. I updated the files so that you get to know it's implementation. Since it is working we want to replace it with some other loader.

**Pull request closed without merging**